docs/serving/samples/cloudevents/cloudevents-nodejs/Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM registry.access.redhat.com/ubi8/nodejs-12
```

Comment:
Was not able to use `node:12-slim` as I'm using a nodejs package using github url, and base image needs git.